src/RequestParser.php

```diff
@@ -193,6 +208,13 @@ private function applySorter(Builder $builder, array $sorter)
         return $builder;
     }
 
+    private function applyPaginator(Builder $builder, array $paginator): Builder
+    {
+        if ($paginator['limit'])
```

Comment:
Referring to https://www.php-fig.org/psr/psr-12/#51-if-elseif-else, 
can you add curly braces after "IF" statement ?